doc/content/getting-started/users-orgs/cli/org/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Organization Management"
+description: ""
+---
+
+This section contains instructions for managing organizations with the CLI.
+
+<!--more-->
+
+## Creating Organizations
+
+Administrators can create organizations as follows:
+
+```bash
+$ ttn-lw-cli organizations create org1 --user-id=user1 \
+  --name "My Colleague" \
```

Comment:
This implies the organization is a colleague, which is not the right cohesion

Reply:
true, I didn't think about that. It shouldn't break them yet since they are still versioned, but from 3.9 on